qiskit/circuit/library/data_preparation/pauli_feature_map.py

```diff
@@ -117,6 +118,7 @@ def __init__(self,
         Args:
             feature_dimension: Number of qubits in the circuit.
             reps: The number of repeated circuits.
+            alpha: The Pauli rotation factor
```

Comment:
Can you describe this a little better? E.g. explicitly say that this is a _multiplicative_ factor in the pauli rotations